doc/INSTALL.md

```diff
@@ -53,6 +53,45 @@ bitcoind &
 ```
 **Note**: You may need to include `testnet=1` in `bitcoin.conf`
 
+
+To Build on Fedora 27 or above
+---------------------
+
+Make sure you have [bitcoind](https://github.com/bitcoin/bitcoin) available to run
+
+Get dependencies:
+```
+$ sudo dnf install -y autoconf automake gcc git libtool gmp-devel libsq3-devel python2-devel python3-devel net-tools libsodium-devel
```

Comment:
Seems this is the only line that is fedora specific?

Reply:
yes, want me to put in the main page as well with ubuntu notes or change something?

Reply:
Thanks for testing @cdecker. Strange, just tested here with a fresh install from Fedora-Server-dvd-x86_64-27-1.6.iso, libtool was available. What's the error output are you facing? Which fedora version docker image is using?

Reply:
I was using `fedora:latest` which seems to be version 27:

```
# cat /etc/fedora-release
Fedora release 27 (Twenty Seven)
```

Reply:
Since the rest of the instructions are identical for all distros I'm tempted to just make the dependency step a list with an item for each distro.

Reply:
@cdecker Concept ACK